cloud/misc/proxmox_kvm.py

```diff
@@ -604,6 +604,19 @@
     name        : spynal
     node        : sabrewulf
     state       : current
+
+# Clone VM from template ( or another VM )
+- proxmox_kvm:
+       api_user    : tfm@pve
+       api_password: secret
+       api_host    : helldorado
+       clone       : 555
```

Comment:
The `vmid` and `newid` is required in your workflow, i hope, the source identifier can use both of `name` or `vmid` .  Cf `get_vmid` and `get_vm` function.